test-jest/cartesian/Brush.spec.tsx

```diff
@@ -36,11 +41,13 @@ describe('<Brush />', () => {
 
   test('Render panorama when specified LineChart as child', () => {
     const { container } = render(
-      <Brush x={100} y={50} width={400} height={40} data={data}>
-        <LineChart>
-          <Line />
-        </LineChart>
-      </Brush>,
+      <BarChart width={400} height={100} data={data}>
```

Comment:
any reason this is being wrapped in a bar chart?

Reply:
Because Brush render SVG elements in the page. 
If not wrapped in a <svg /> they aren't valid element for the browser.

<img width="1677" alt="image" src="https://user-images.githubusercontent.com/24919330/214223331-3beb83f9-93c2-4eab-9471-18247cbbda1d.png">

If you try this out in the demo page you see that there isn't any visible Brush in the page.

<img width="1530" alt="image" src="https://user-images.githubusercontent.com/24919330/214223815-ee8bd2db-21da-466e-a7e3-25b11f85a0e8.png">

If you prefer I can wrap them in a Surface like in `demo/component/BrushDemo.tsx`:

```tsx
<Surface width={800} height={200}>
          <Brush {...props} />
</Surface>
```

Reply:
I think we should keep these simple and yeah wrap in `Surface` instead of an entire chart